packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -81,8 +81,8 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
     bool internal initialized;
 
     /// @notice Semantic version.
-    /// @custom:semver 0.1.0
-    string public constant version = "0.1.0";
+    /// @custom:semver 0.1.1
+    string public constant version = "0.1.1";
```

Comment:
nit: this is a minor bump, changes bytecode.

(We haven't been following semver rules, but should start now that we're on a minor :) )